client/state/login/selectors.js

```diff
@@ -70,21 +62,15 @@ export const getTwoFactorPushPollInProgress = state => get( state, 'login.twoFac
 export const getTwoFactorPushPollSuccess = state => get( state, 'login.twoFactorAuthPushPoll.success', false );
 
 /**
- * True if two factor authentication is enabled for the logging in user.
- * False if not, null if the information is not available yet.
+ * Determines whether two factor authentication is enabled for the logging in user.
  *
  * @param  {Object}   state  Global state tree
- * @return {?Boolean}        Whether 2FA is enabled
+ * @return {Boolean}        Whether 2FA is enabled
  */
 export const isTwoFactorEnabled = ( state ) => {
 	const twoFactorAuth = get( state, 'login.twoFactorAuth' );
-	if ( ! twoFactorAuth ) {
-		return null;
-	}
 
-	return (
-		twoFactorAuth.user_id !== ''
-	);
+	return ! isEmpty( twoFactorAuth );
```

Comment:
Actually I was thinking about removing it as well, since if we got two factor enabled, we'll have this reducer populated, and if not it'll be empty, so checking for `user_id` is redundant IMO, so I agree here with @stephanethomas